xml/System.Data.Objects/ObjectQuery`1.xml

```diff
@@ -476,9 +476,9 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:System.Data.Objects.ObjectQuery`1.OfType``1*> is used to filter query results by a specific entity or complex type. This supports a conceptual model with object inheritance. For more information, see [Entity Data Model Inheritance (Application Scenarios)](http://msdn.microsoft.com/library/5de40bf8-527f-4f2e-96e3-61164f429caa). The  
+ <xref:System.Data.Objects.ObjectQuery`1.OfType*> is used to filter query results by a specific entity or complex type. This supports a conceptual model with object inheritance. For more information, see [Entity Data Model Inheritance (Application Scenarios)](http://msdn.microsoft.com/library/5de40bf8-527f-4f2e-96e3-61164f429caa). The  
```

Comment:
Unrelated to your PR, the link to MSDN goes to documentation that is tagged as pre-release and is orphaned. It might be best to replace the link with a link to [Entity Data Model: Inheritance](~/docs/framework/data/adonet/entity-data-model-inheritance.md).